mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -63,6 +73,24 @@ func runSeedlistDiscoveryDirectory(mt *mtest.T, subdirectory string) {
 	}
 }
 
+// runSeedListDiscoverPingTest will create a new connection using the test URI and attempt to "ping" the server.
+func runSeedListDiscoveryPingTest(mt *mtest.T, clientOpts *options.ClientOptions) {
```

Comment:
From the [Go documentation](https://pkg.go.dev/testing#T.Helper) on test helpers: 

> Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Helper may be called simultaneously from multiple goroutines.

I don't think I would consider this function a helper, as it's actually running a test and we want line information to print. I put it into it's own function because I thought that it read a little better. But maybe it would be better to put all of the logic in "runSeedlistDiscoveryTest". What are your thoughts?

Reply:
The extra function seems fine to me, and you're right that adding `mt.Helper()` is probably not a good idea, as `runSeedListDiscoveryPingTest` contains testing assertions.